onboard/src/pages/TestEmbeddedSigningPage.css

```diff
@@ -0,0 +1,20 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  overflow: hidden;
+}
+
+iframe {
```

Comment:
Could this iframe code be affecting the youtube embed on the front page? It seemed to have shrunk in size when we looked at it on your screen. 

Can you add something more specific for this iframe code, like a class that affects specifically your page? Same issue for changing body,html. This may affect other pages.